scripts/split_libraries_fastq.py

```diff
@@ -151,6 +161,24 @@
 
 def main():
     option_parser, opts, args = parse_command_line_parameters(**script_info)
+    read_arguments_from_file = opts.read_arguments_from_file
+
+    # these arguments can optionally be read from a file, reasoning is to
+    # allow arguments that would span over hundreds of samples and would be
+    # prohibitive to execute as a command line call
+    if read_arguments_from_file:
+        read_f = lambda x: open(x, 'U').read().strip().split('\n')
```

Comment:
would it be better to actually create this function using `def` so that you can use a context manager with `open`?